CMakeLists.txt

```diff
@@ -373,6 +373,13 @@ if (NOT S2N_EXECINFO_AVAILABLE)
 endif()
 feature_probe_result(S2N_STACKTRACE ${S2N_STACKTRACE})
 
+if (S2N_BRANCH_ALIGN AND NOT S2N_FUZZ_TEST)
+    set(BRANCH_ALIGN_FLAG "-Wa,-mbranches-within-32B-boundaries")
+    target_compile_options(${PROJECT_NAME} PRIVATE ${BRANCH_ALIGN_FLAG})
+else()
+    set(BRANCH_ALIGN_FLAG "")
```

Comment:
nit: Is the `BRANCH_ALIGN_FLAG` variable needed? This might be simpler:
```suggestion
    target_compile_options(${PROJECT_NAME} PRIVATE "-Wa,-mbranches-within-32B-boundaries")
```